src/js/puzzles.js

```diff
@@ -291,19 +291,19 @@ var puzzlesArray = [
   id: 42,
   sentence: 'Learn how to strive a truck.',
   word: 'strive',
-  hint: 'you need a license to do this',
+  hint: 'need a license to do this',
   answer: 'drive'
 },
 {
   id: 43,
-  sentence: 'It is better to practice with a utensil',
+  sentence: 'It is better to practice with a utensil.',
   word: 'utensil',
   hint: 'erasable',
   answer: 'pencil'
 },
 {
   id: 44,
-  sentence: 'Pursuit of sappiness.',
+  sentence: 'A pursuit of sappiness.',
```

Comment:
Here, this is a collocation (a common phrase), so putting an indefinite article on it changes the meaning significantly. If we want an article here, let's use `the` instead.

Reply:
Fixed on my latest commit.